ghost/core/core/server/services/mega/template.js

```diff
@@ -70,6 +71,10 @@ table {
     width: 100%;
 }
 
+table.inline {
+  width: auto !important;
+}
+
```

Comment:
@lenabaidakova Quick check, this CSS doesn't has any side-effects on existing template if the alpha flag is off right ?

Reply:
It shouldn't but you are right, better to isolate them =)
Moved to feedback buttons https://github.com/TryGhost/Ghost/pull/15619/commits/075e753af78efec452c5ce012c39b47f41ed21b7